### PR TITLE
feat : homepage

### PR DESCRIPTION
**Pull request by sudipkundu999**
## [Please review these files only](https://github.com/sudipkundu999/CobraTube/pull/1/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+      <Header />
+      <div className="app">
+        <Sidebar />
+        <Routes>
+          <Route path="/" element={<Homepage />} />
+          {/* <Route element={<AuthRoutes />}>
+          <Route path="/login" element={<Login />} />
+          <Route path="/signup" element={<Signup />} />
+        </Route> */}
+          {/* <Route element={<RequiresAuth />}> */}
```
Comment (DivyanshSareen) [inline src/App.js:24]: remove comments if not intended for future use

File: src/components/header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          </Link>
+        </div>
+      </div>
+      <input
+        // value={}
+        type="text"
+        placeholder="Search"
+        className="nav-search search-mobile"
+        // onChange={(e) =>}
+      />
```
Comment (DivyanshSareen) [inline src/components/header/Header.jsx:43]: Can use aria-label or some other [hidden label approach](https://www.w3.org/WAI/tutorials/forms/labels/) for accessibility.

**Review by DivyanshSareen — APPROVED** [review]
Everything looks great. Good to merge

File: src/App.js (hunk 1 above)
Comment (sudipkundu999) [reply src/App.js:24]: These are for future use

File: src/components/header/Header.jsx (hunk 2 above)
Comment (sudipkundu999) [reply src/components/header/Header.jsx:43]: Thank you for sharing this. This was new to me.

File: src/App.js [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
 function App() {
-  return <h1>CobraTube</h1>;
+  return (
+    <div>
+      <Header />
+      <div className="app">
+        <Sidebar />
+        <Routes>
+          <Route path="/" element={<Homepage />} />
+          {/* <Route element={<AuthRoutes />}>
```
Comment (itsmebipul00) [inline src/App.js:20]: You could remove commented code

File: src/index.css [hunk 4]
```diff
@@ -2,5 +2,10 @@
 @import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css");
 
 body {
-  background-color: var(--white-light);
+  background-color: #162538;
+  color: white;
```
Comment (itsmebipul00) [inline src/index.css:6]: You could use variables here

**Review by itsmebipul00 — APPROVED** [review]
Awesome work, good to merge

File: src/index.css (hunk 4 above)
Comment (sudipkundu999) [reply src/index.css:6]: The color is used just once in the code so I used it directly

**Pull request merged**